README.md

```diff
@@ -26,3 +26,37 @@ While already dated (in particular, the four modules dune-xt-common, dune-xt-gri
     T. Leibner and R. Milk and F. Schindler: "Extending DUNE: The dune-xt modules"
     Archive of Numerical Software, 5:193-216, 2017
     https://www.doi.org/10.11588/ans.2017.1.27720
+
+
+Interactive development of Python bindings
+------------------------------------------
+
+We recommend running the preconfigured docker environment in a [rootless docker setup](https://docs.docker.com/engine/security/rootless/).
```

Comment:
I would prefer not to advertise your setup as the general use one. AFAICT this is not based on `deps/Dockerfile` / the compose setup, right? So that would basically mean separate infra to maintain, which is not tested in CI. I also feel like you create script tooling around docker that would already be covered by docker-compose.
I would much prefer if the advertised general use would utilize the existing infra. And not tell people to run a IDE inside a container.
One solution could be shipping a `devcontainer.json` for vscode that points to the compose/dockerfile.

Reply:
Agreed. This arose from the need to have something quick, it was there and working.
- it's advertised as the general one, because the other modes that we prefer are missing yet (I'd imagine this to be the last one in the list)
- the entrypoint I use corresponds to the script that is used to start the container, and is targeted towards interactive use of the container. One option could be to derive from whats in `deps/Dockerfile` (instead of `debian-qtcreator`), or even to check if my entrypoint would cover the other case es well
- what I like about the script is that it's orthogonal to containerisation and can also be used without
- what I don't like about that setup is the reliance on the `PATH.sh`

Reply:
- Isn't the script the used in the docker workflow? So it's exactly not-orthogonal to it? 
- I think what you need is a build-arg to the `deps/Dockerfile` that lets you optionally mixin your qtcreator setup. 

My position is still that nobody who isn't familiar with this kind of workflow already should learn it now. Remote/Containerized development workflows are much simpler to use for new users, since they have much less contact surface with the container innards.